### PR TITLE
Add ohrwurm interface

### LINKED ISSUE #1: There should be a viewer interface for ohrwurm microservice
**Describe the feature or change you'd like**
The plan is to implement following functionality:
Service Login

### Registry
Adding multiple ohrwurms should be possible by registering them into a
local service registry file. Later there should be external registry service.

For example:
```json
{
  "ohrwurm": [
    {"url": "ms1.ohrwurm.snek.at/graphql", "options": []}
  ]
}
```
Further specification can be applied when implementing this feature.

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 

-   [x] Have you added an explanation of what your changes do and why you'd like them to be included?
-   [ ] Have you updated or added documentation for the change?
-   [x] Have you tested your changes with successful results?

**Type of Changes:**

 

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Documentation (non-breaking change which adds documentation)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- 

**What is the new behavior (if this is a feature change)?**
- Now there is a general structure for possible future Microservices.
- Nowt there is a interface for viewing projects and tracks of a ohrwurm. Closes #1

**Other information**:
image

### CODE REVIEW

**Review by neuwirte — DISMISSED** [review]
lgtm

File: src/components/molecules/Login/index.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <div>
+      <MDBContainer>
+        <MDBRow className="justify-content-center">
+          <MDBCol md="6">
+            <MDBCard>
+              {/* <div className="header pt-3 grey lighten-2">
+                  <MDBRow className="d-flex justify-content-start">
+                    <h3 className="deep-grey-text mt-3 mb-4 pb-1 mx-5"></h3>
+                  </MDBRow>
+                </div> */}
```
Comment (kleberbaum) [inline src/components/molecules/Login/index.tsx]:
```suggestion
```

File: src/components/molecules/Login/login.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    box-shadow: 0 1px 0 0 $snekGreen !important;
+  }
+  input[type="password"]:focus:not([readonly]) + label {
+    color: $snekGreen !important;
+  }
+  // input[type="text"]:focus {
+  //   box-shadow: 0 1px 0 0 green !important;
+
+  //   border-bottom: 1px solid green !important;
+  // }
```
Comment (kleberbaum) [inline src/components/molecules/Login/login.scss]:
```suggestion
```

File: src/components/organisms/services/ohrwurm/index.tsx [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+              entries={
+                this.props.ohrwurm.tracks?.items
+                  ? this.props.ohrwurm.tracks.items
+                  : []
+              }
+              onTranscriptClick={this.selectTrack}
+            ></TrackTable>
+          </>
+        )}
+        {/* */}
```
Comment (kleberbaum) [inline src/components/organisms/services/ohrwurm/index.tsx]:
```suggestion
```

File: src/components/organisms/services/ohrwurm/index.tsx [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+                onClick: () => this.switchTable("PROJECT"),
+              },
+              {
+                name: "Track",
+                active: true,
+              },
+            ]}
+          ></Breadcrumbs>
+        )}
+
```
Comment (kleberbaum) [inline src/components/organisms/services/ohrwurm/index.tsx]:
```suggestion
```

File: src/components/organisms/services/ohrwurm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+ * Provides its connected component with the pieces of the data it needs from
+ * the store, and the functions it can use to dispatch actions to the store.
+ *
+ * Got access to the history object’s properties and the closest
+ * <Route>'s match.
+ */
+export default withRouter(
+  connect(mapStateToProps, mapDispatchToProps)(Ohrwurm)
+);
+//#endregion
```
Comment (kleberbaum) [inline src/components/organisms/services/ohrwurm/index.tsx:231]:
```suggestion
//#endregion

```

File: src/components/organisms/services/ohrwurm/ohrwurm.scss [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.modal-body {
+  overflow-y: auto;
+}
+
+// .active-pink-2 input[type="text"]:focus:not([readonly]) {
+//   border-bottom: 1px solid #f48fb1;
+//   box-shadow: 0 1px 0 0 #f48fb1;
+// }
```
Comment (kleberbaum) [inline src/components/organisms/services/ohrwurm/ohrwurm.scss]:
```suggestion
```

File: src/store/actions/ohrwurmActions.ts [hunk 7]
```diff
@@ -0,0 +1,232 @@
...
+      dispatch({
+        type: "OHRWURM_FETCH_TRACKS_FAILURE",
+        payload: {
+          errorDetails: serializeError(ex),
+        },
+      });
+    }
+  };
+};
+
```
Comment (kleberbaum) [inline src/store/actions/ohrwurmActions.ts]:
```suggestion
```

File: src/store/reducers/index.ts [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+//#region > Config
+const rootReducer = combineReducers({
+  // Loading bar
+  loadingBar: loadingBarReducer,
+  // User data
+  user: userReducers,
+  // Ohrwurm
+  ohrwurm: ohrwurmReducer,
+});
+
```
Comment (kleberbaum) [inline src/store/reducers/index.ts]:
```suggestion
```

**Review by kleberbaum — DISMISSED** [review]
Should be ok to merge.

**Pull request merged**